controllers/dspipeline_controller.go

```diff
@@ -20,7 +20,7 @@ import (
 	"context"
 	"fmt"
 	"sigs.k8s.io/controller-runtime/pkg/controller"
-	"time"
+	// 	"time"
```

Comment:
This comment can be removed.

```suggestion
```

Reply:
Good catch! My bad I didn't realize I hadn't taken this out. Deleted the comment.